src/app/reactive-templates/reactive-components/components/button/button.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, computed, forwardRef, inject } from "@angular/core";
+import { defineParameters, Parameter } from "../../parameters";
+import { ROW_PARAMETERS, RowBaseComponent } from "../../row-base.component";
+import { CommonModule } from "@angular/common";
+import { IonicModule } from "@ionic/angular";
+import { TemplateTranslateService } from "src/app/shared/components/template/services/template-translate.service";
+import { TemplatePipesModule } from "src/app/shared/components/template/pipes";
+import { RowListComponent } from "../../row-list.component";
+
+const parameters = () =>
```

Comment:
This new parameters syntax is obviously a great improvement. It'd be worth comparing/contrasting with the proposed implementation in #3096, which accomplishes something very similar, and deciding which we want to use